app/app.html

```diff
@@ -7,7 +7,7 @@
 <body>
   <div id="app">
     <!-- <webview id="androidMessagesWebview" src="https://davidwalsh.name/demo/notifications-api.php" autosize></webview> -->
-    <webview id="androidMessagesWebview" src="https://messages.android.com/" autosize preload="bridge.js"></webview>
+    <webview id="androidMessagesWebview" src="https://messages.google.com/web" autosize preload="bridge.js"></webview>
```

Comment:
You may also need to change these:

- https://github.com/chrisknepper/android-messages-desktop/blob/master/src/app.js#L30
- https://github.com/chrisknepper/android-messages-desktop/blob/master/src/app.js#L38
- https://github.com/chrisknepper/android-messages-desktop/blob/master/src/helpers/webview/context_menu.js#L77

 since it is looking for the old URL also.

This was just a quick search and may not be required for functionality; however, it is important for consistency. Also, I may have skipped some references.

Reply:
I’ve done this now (though it’s in three separate pull requests cos I’m a failure at github)